receiver/prometheusreceiver/internal/metrics_adjuster.go

```diff
@@ -201,73 +200,56 @@ func NewMetricsAdjuster(tsm *timeseriesMap, logger *zap.Logger) *MetricsAdjuster
 	}
 }
 
-// AdjustMetrics takes a sequence of metrics and adjust their values based on the initial and
-// previous points in the timeseriesMap. If the metric is the first point in the timeseries, or the
-// timeseries has been reset, it is removed from the sequence and added to the timeseriesMap.
-// Additionally returns the total number of timeseries dropped from the metrics.
+// AdjustMetrics takes a sequence of metrics and adjust their start times based on the initial and
+// previous points in the timeseriesMap.
+// Returns the total number of timeseries that had reset start times.
 func (ma *MetricsAdjuster) AdjustMetrics(metrics []*metricspb.Metric) ([]*metricspb.Metric, int) {
 	var adjusted = make([]*metricspb.Metric, 0, len(metrics))
-	dropped := 0
+	reset := 0
```

Comment:
Wouldn't it be better to call this resets?

Reply:
I think I was maintaining tense from `dropped`.  "Number of metrics dropped" => "Number of metrics reset".  That said, it's easy to rename.